Hashlink.js

```diff
@@ -137,4 +176,29 @@ export class Hashlink {
     this.registeredTransforms[transform.algorithm] = transform;
   }
 
+  /**
+   * Finds a registered decoder for a given set of bytes or throws an Error.
+   *
+   * @param {Uint8Array} bytes - A stream of bytes to use when matching against
+   *   the registered decoders.
+   */
+  _findDecoder(bytes) {
+    const transforms = Object.values(this.registeredTransforms);
+    const decoder = transforms.reduce((decoder, transform) => {
+      let match = true;
+      let index = 0;
+      while(match && (index < transform.identifier.length)) {
+        match = transform.identifier[index] === bytes[index];
+        index++;
+      }
+      return (match) ? transform : decoder;
```

Comment:
```suggestion
      return match ? transform : decoder;
```

Reply:
Fixed in d31f47399a9014b2a958d0c7494196a5a7f0bc38.